src/components/atoms/UserStatusDropdown/UserStatusDropdown.tsx

```diff
@@ -52,9 +54,11 @@ export const UserStatusDropdown: React.FC<UserStatusDropdownProps> = ({
         {userStatus.status}&nbsp;
       </div>
       {showDropdown && (
-        <DropdownButton id="user-status-dropdown" title={"change status"}>
-          {userStatusDropdownOptions}
-        </DropdownButton>
+        <Dropdown
+          id="user-status-dropdown"
```

Comment:
Why the need for `id` prop?

Reply:
Well, based on docs from Bootstrap https://react-bootstrap.github.io/components/dropdowns/#dropdown-button-props

> An html id attribute for the Toggle button, necessary for assistive technologies, such as screen readers.

But we can avoid it. 
fixed in dc3dcc064168e6d2acbd386f35b99fa9f36961a1

Reply:
We can remove the `id` prop **and** still provide id for the Bootstrap component if it needs it for
>necessary for assistive technologies, such as screen readers.

☝️ we can do that by using `generateId` inside the component if/when that id is needed to link it with another (like `label` with `input`)

Reply:
... or maybe even for some 3rd party script (if that's how Bootstrap uses the id property of `Dropdown`)